### DISCUSSION TITLE
how to get the nick to appear in chat in stead of displayname

### DISCUSSION THREAD

**Discussion by threlief** [EssentialsX/Essentials#4096] [Q&A] [answered]
i forgot how to set it up so that peoples /nick name appears in chat instead of the display name

**Comment by threlief** [comment]
'{PREFIX}{NICKNAME}: {MESSAGE}' this what i put but now everyone has {NICKNAME) as their name :/

**Comment by triagonal** [comment, marked as answer]
`{NICKNAME}` didn't exist at the time this discussion was posted, but now it does! `{NICKNAME}` has been added as a chat format placeholder with #4098. The placeholder will show the player's EssentialsX nickname. If the player doesn't have a nickname, it will show the player's username.

Update to the [latest version of EssentialsX](https://essentialsx.net/downloads.html) to use the new placeholder.
